osu.Game/Online/Chat/MessageFormatter.cs

```diff
@@ -246,6 +246,44 @@ public static LinkDetails GetLinkDetails(string url)
             return new LinkDetails(LinkAction.External, url);
         }
 
+        public static string? GetUrl(LinkDetails link)
+        {
+            switch (link.Action)
+            {
+                case LinkAction.External:
+                    return (string)link.Argument;
+
+                case LinkAction.OpenBeatmap:
+                    return $@"https://{WebsiteRootUrl}/beatmaps/{link.Argument}";
+
+                case LinkAction.OpenBeatmapSet:
+                    return $@"https://{WebsiteRootUrl}/beatmapsets/{link.Argument}";
+
+                case LinkAction.OpenUserProfile:
+                    return $@"https://{WebsiteRootUrl}/users/{link.Argument}";
```

Comment:
This doesn't work on the beatmap wedge:
![osu!_jCKABii2Zg](https://github.com/ppy/osu/assets/35318437/df3802c1-4561-4fe2-ac01-23eaf7b45b9e)